crates/sui-indexer/src/schema/mod.rs

```diff
@@ -64,3 +64,11 @@ pub use inner::tx_digests;
 pub use inner::tx_input_objects;
 pub use inner::tx_recipients;
 pub use inner::tx_senders;
+
+// Postgres only tables
```

Comment:
Added these because otherwise when the compatibility macro tries to look for these tables, it will fail, hopefully that's okay!